marklogic-data-hub-central/ui/e2e/cypress/integration/explorer/scenarios.json.tsx

```diff
@@ -153,17 +167,14 @@ xdescribe('json scenario for table on browse documents page', () => {
     cy.visit('/');
     cy.contains(Application.title);
     cy.loginAsDeveloper().withRequest();
-    // temporary change as tile is not working
-    homePage.getTitle().click();
-    cy.wait(500);
-    // temporary change end here
-    homePage.getBrowseEntities().click();
-    cy.wait(2000);
-    browsePage.clickTableView();
-    browsePage.selectEntity('All Entities');
+    cy.location('pathname', { timeout: 10000 }).should('include', '/tiles');
+    cy.wait(200);
+    cy.waitUntil(() => toolbar.getExploreToolbarIcon()).click();
+    cy.waitUntil(() => browsePage.getExploreButton()).click();
+    browsePage.getTableView();
```

Comment:
@timur-isangulov the default view is table so we don't need this line to test table scenarios

Reply:
Removed. Thanks.